src/plugins/vis_type_timeseries/public/application/components/annotations_editor.js

```diff
@@ -131,9 +132,13 @@ export class AnnotationsEditor extends Component {
                   label={
                     <FormattedMessage
                       id="visTypeTimeseries.annotationsEditor.indexPatternLabel"
-                      defaultMessage="Index pattern (required)"
+                      defaultMessage="Index name (required)"
```

Comment:
Did we check this with UX team?

Reply:
Yes, after some investigation with UX, they preferred using `Index name` since users can query with regex like patterns on index names.